packages/lib/uuid.ts

```diff
@@ -8,6 +8,8 @@ import { customAlphabet } from 'nanoid/non-secure';
 // > indefinitely
 const nanoid = customAlphabet('0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz', 22);
 
+const createUuidgenCustomAlphabet = (length: number) => customAlphabet('0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz', length);
```

Comment:
Since uuidgen is frequently used, could we use the same method as above and cache the output of `customAlphabet()`? In this particular case it means you'll need to cache a different version for each `length`

Reply:
I end up creating a cache object for the different argument calls, do you think this is better?

https://github.com/laurent22/joplin/blob/10aea881834276edf75bb73e800e93b612b1f444/packages/lib/uuid.ts#L24-L39

Reply:
I did this because uuidgen sometimes is called with other arguments besides the default one.

Reply:
Yes thank you, this implementation with a different cache for each length definitely makes sense